src/stories/textInput/textInput.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "./textInput.module.css";
+
+export default function TextInput() {
+  return (
+    <form>
+      <input type={"text"} className={styles.textInput}></input>
```

Comment:
Just an idea as extension: Maybe you could set a placeholder, so that the user can see that this is an input field.